app/decorators/scanned_resource_decorator.rb

```diff
@@ -20,6 +20,21 @@ def metadata_adapter
     Valkyrie.config.metadata_adapter
   end
 
+  def rights_statement
```

Comment:
I recently added a `rights_statement_uri` method to Plum to make it easier to access the URI instead of the HTML rendering.  Is that the best way to allow access to both forms?  Or would it make more sense to have this have a different name like `rendered_rights_statement`?